docs/setup/installation.md

```diff
@@ -18,19 +18,177 @@ that your email address is probably `user@example.com` rather than
 
 ## Installing Synapse
 
-### Installing from source
+### Prebuilt packages
+
+Prebuilt packages are available for a number of platforms. These are recommended
+for most users.
+
+#### Docker images and Ansible playbooks
+
+There is an official synapse image available at
+<https://hub.docker.com/r/matrixdotorg/synapse> which can be used with
+the docker-compose file available at
+[contrib/docker](https://github.com/matrix-org/synapse/tree/develop/contrib/docker).
+Further information on this including configuration options is available in the README
+on hub.docker.com.
+
+Alternatively, Andreas Peters (previously Silvio Fricke) has contributed a
+Dockerfile to automate a synapse server in a single Docker image, at
+<https://hub.docker.com/r/avhost/docker-matrix/tags/>
```

Comment:
Hadn't seen this syntax before. I thought it'd make markdown sad because I think you can embed HTML in markdown? But apparently [it's A Thing in GFM](https://github.github.com/gfm/#autolinks).

Reply:
FWIW: I don't think this is GFM-specific. (indeed, CommonMark seems to have it for example).

It's commonly used for e-mail addresses in my experience, probably because e-mail itself formats things like `From: Example <example@example.org>`; I would guess `<..>` carried through as a plaintext convention that wound up in Markdown?